### DISCUSSION TITLE
Field validation upon creation

### DISCUSSION THREAD

**Discussion by balbern** [directus/directus#7241] [Archive]
As of now validation is only applied when updated.
We need to apply the validation rules also when the item is created.
This issue is mostly prevalent for the "id" column, which can only be validated upon creation, since it can not be updated.

**Comment by benhaynes** [comment]
Are you saying you want to add this feature, or the existing feature is broken? If I go to:

`Settings > Permissions > [Collection] + [CREATE] > Use Custom`

Then I get an option for Create Validation Rules (see below)... is this what you're referring to?

![Screen Shot 2021-08-06 at 8 45 16 AM](https://user-images.githubusercontent.com/522079/128512546-25ba9ff7-16b1-4a9e-96f0-7ccbfbb2970d.png)

**Reply by balbern** [reply]
Okay, makes perfect sense, now.
I was writing validation only in the "edit/update" validations, and of course they did not apply
I am currently trying around and I did not think of checking for that option in the creation permissions.
